auth_signup_verify_email/controllers/main.py

```diff
@@ -70,10 +77,16 @@ def passwordless_signup(self, values):
         except Exception as error:
             # Duplicate key or wrong SMTP settings, probably
             _logger.exception(error)
-
-            # Agnostic message for security
-            qcontext["error"] = _(
-                "Something went wrong, please try again later or contact us.")
+            if request.env["res.users"].sudo().search(
+               [("login", "=", qcontext.get("login"))]):
+                qcontext["error"] = _(
+                    "Another user is already registered using this email"
+                    " address.")
```

Comment:
About the subject of security through obscurity, I'm generally against it.

You can see, @pedrobaeza, that I just mentioned your GitHub login name, so if I wanted to hack on your account I already know that; when I send you an email, I know your Google login name; when I read your tweets, I know your twitter login; etc.

FTR:

![captura de pantalla de 2018-12-05 08-29-34](https://user-images.githubusercontent.com/973709/49500311-50c25380-f868-11e8-8d63-3aa9a7b59d35.png)

If you want actual security for your user accounts, there are other modules for that, so this is :+1:  IMHO.